### ISSUE TITLE
Add a counter for the number of profile cards in the website

### ISSUE THREAD

**Issue by Ninjavin** [dsc-iem/WebDev-Hacktoberfest#1]
Include a sticky counter in the bottom right which shows the number of profile cards in the website.

**Comment by VaibhaveS** [comment]
Hey, can you assign me this issue?

**Comment by aditimaurya** [comment]
Hey! I would like to work on this if you haven't assign this issue to anyone. Can you please provide me with more details regarding the issue ?

**Comment by Ninjavin** [comment]
Hey @VaibhaveS and @aditimaurya thank you for showing interest in resolving the issue. As it is reserved for Hacktoberfest, claim it after 1st October.

**Comment by VaibhaveS** [comment]
@Ninjavin you can assign the issue whenever you want, we can do a PR after OCT 1.

**Comment by aditimaurya** [comment]
@Ninjavin yeah, you can assign this issue to any one of us because hacktoberfest is on the way and it would be good if you assign this as soon as possible. We will make our PR'S after that.

**Comment by Ninjavin** [comment]
@aditimaurya I am assigning this issue to you. https://www.figma.com/file/88M7Vx2bWgJ3eUdNO12NxE/Untitled?node-id=0%3A1
This wil give you an idea of the counter I want. You may improvise it, and do the needful changes.

**Comment by aditimaurya** [comment]
okay sure. I'll be looking into this issue. Thanks @Ninjavin

**Comment by aditimaurya** [comment]
@Ninjavin can I fork this repo and start making changes? I'll make a PR in October, will it be fine?

**Comment by Ninjavin** [comment]
Yes @aditimaurya . Definitely.

**Comment by ommiy2j** [comment]
can you assign me this issue.......

**Comment by rajayush213** [comment]
Hey @Ninjavin ! Can you assign this issue to me?

**Comment by peekaboo5149** [comment]
Hi @Ninjavin  can you pls assign me this issue

**Comment by diksha0399** [comment]
hi, @Ninjavin , i want to contribute to this repo, so please could you assign me this issue.

**Comment by VaibhaveS** [comment]
@diksha0399 it has already been assigned to @aditimaurya

**Comment by Ninjavin** [comment]
@ommiy2j @rajayushGithub @peekaboo5149 @diksha0399 
The issue is currently assigned to @aditimaurya 
You may create new issues you want to work on

**Comment by aditimaurya** [comment]
@Ninjavin is it compulsory to ad the kind of cards you have mentioned in that website? or I can modify them? Also can you please tell me in which HTML file should I make changes? there are two different files,.

**Comment by Ninjavin** [comment]
If you want to change the CSS, you can open a different issue and work on it. Make a separate JS file to add counter.

**Comment by aditimaurya** [comment]
@Ninjavin  I/m facing some problems, could you please help me out? I am not sure whether I've to just make a separate file for the counters which is independent of the other codes (using Js ) or I have to you know, make a dependent counter which is specifically for this particular website. I mean, it's a little hard to figure out how to change the already written code and where to write my code in the large file, so could you please help me out? like please tell me on which file I need to work or I have to make a complete independent js file for the counter only.
